apps/docs/src/packages/headless.md

```diff
@@ -0,0 +1,58 @@
+---
+outline: [2, 3]
+---
+
+<script lang="ts" setup>
+import packageJson from "../../../../packages/headless/package.json";
+</script>
+
+# @sit-onyx/headless
+
+::: warning Work in progress / Active development
+This library is currently in early / active development.
+:::
+
+{{ packageJson.description }}.
+
+Inspired by [Melt UI](https://melt-ui.com).
+
+## Installation
+
+<!--
+Make sure that this chapter is kept up to date with installation steps in
+packages/headless/README.md file.
+ -->
+
+Install the npm package with your corresponding package manager:
+
+::: code-group
+
+```sh [pnpm]
+pnpm add sit-onyx/headless
+```
+
+```sh [npm]
+npm install sit-onyx/headless
+```
+
+```sh [yarn]
+yarn install sit-onyx/headless
+```
+
+:::
+
+## Composables
+
+### useComboBox
+
+```vue
+<script lang="ts" setup>
+import { createCombobox } from "@sit-onyx/headless";
+
+const {
+  elements: {
+    ...
+  }
+ } = createCombobox();
+</script>
+```
```

Comment:
isn't a <template ... missing to fully showcase the usage?

Reply:
Yes but I don't know how Jonathan intended the composable to work :D I think its just a placeholder, thats why I added the "Work in progress" warning at the top